WalletWasabi.Documentation/Guides/Wasabi2TestVectors.md

```diff
@@ -0,0 +1,161 @@
+
+- Set limit (in wallet settings) above your balance
+- Click Play
+- CoinJoin should be on hold - Pleb stop preventing it
+- Set limit below your balance
+- Wait - CoinJoin should start automatically at maximum in 1 minute
+
+#### CoinJoin threshold (PlebStop) - stopping
+
+- TODO @molnard
+
+### Auto start coinjoin
+
+- Turn on auto start coinjoin
+- Load wallet, see music box is in play
+- Click pause, see coinjoin stops
+- Click play, see coinjoin starts
+- Click pause, restart Wasabi, load same wallet, see that coinjoin is starting
+- Coinjoin to 100% privacy level, see that coinjoin stops, receive fresh bitcoin to the wallet, see that coinjoin is starting
+
+## Send
+
+- Paste all address types [pubkey, script, wrapped segwit, native segwit, taproot, etc]
+- Make a typo in address, see it can't continue
+- Write amount above wallet balance, see it can't continue
+- Write amount of exact wallet balance, see it can continue
+- Write amount below wallet balance, see it can continue
+- Paste BIP21 URI, see it can't change amount
+- Paste BIP21 URI with PayJoin flag, see it can't change amount / address and shows the payjoin logo
+- When payment amount below private wallet balance, see no labels on transaction preview screen
+- When payment amount above private wallet balance, see labels on transaction preview screen, click on edit, change pocket selection, see results
+- Click on privacy optimization, select payment amount higher / lower, see transaction preview amount change
+- Click on the undo button, see transaction preview change
+- Continue and see password box [only if wallet has password], type in wrong password, see it fails, type in correct password, see it broadcast transaction successfully
+
+## History
+
+- See unconfirmed transactions grouped together
+- See multiple coinjoins collapsed into one line item, click expand and see individual coinjoins
+- See transaction details icon pop up when hovering, click it to see details dialog
+- See right click menu options
+- Sort the table by collumn
+
+## Settings
+
+- Change theme to darkmode
+- Turn off Tor, see in status icon that Tor is not connected
+- Try Bitcoin full node integration
+- Turn on / off auto copy / paste, test if it works in receive and send dialogs
+- Turn on start Wasabi on boot
+
+## Close to tray / running in background
+
+- Close Wasabi GUI, see it goes into tray icon
+- Right click tray icon and maximize GUI
+- Right click tray icon and close Wasabi
+
+## Hardware wallets
+
+- Import hardware wallet
+- Generate receive address
+- Check if spend is not possible when hardware wallet not connected
+- Check if spend is possible with connected hardware wallet
+- Check PSBT workflow [coldcard SD card]
+- Check if music box is not visible [coinjoin impossible]
+
+## Watch only wallets
+
+- Import watch only wallet [explained here](https://docs.wasabiwallet.io/FAQ/FAQ-UseWasabi.html#can-i-import-a-watch-only-extended-public-key)
```

Comment:
This requires manual editing of wallet files. I would move this to the advanced Testing section.

Reply:
There's no other way to test watch only wallets, so I guess it's alright.
We can move it to the bottom of the file if needed.

Reply:
I have been thinking a long time ago to add a "watch only" option to the add wallet dialog. I was just not sure if we properly support them...

Reply:
Yes it should be added, but I think that's best done in an future output script descriptor refactoring.

Reply:
Yes, do not spend time on this.